stix_shifter_modules/alertflex/stix_transmission/api_client.py

```diff
@@ -0,0 +1,29 @@
+import base64
+from stix_shifter_utils.stix_transmission.utils.RestApiClient import RestApiClient
+
+
+class APIClient():
+    PING_TIMEOUT_IN_SECONDS = 10
+
+    def __init__(self, connection, configuration):
+        self.endpoint_start = 'alertflex-ctrl/rest/stix-alerts'
+        headers = dict()
+        auth = configuration.get('auth')
+        headers['Authorization'] = b"Basic " + base64.b64encode(
+            (auth['username'] + ':' + auth['password']).encode('ascii'))
+        self.client = RestApiClient(connection.get('host'),
+                                    connection.get('port'),
+                                    connection.get('cert', None),
```

Comment:
we removed this `cert` initialization recently. instead you can use `selfSignedCert` if required. you can remove `cert`.